From 6e74b82835a48e92cda34a17e083f7689676631c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:03:55 -0400
Subject: [powerpc] Fix typo in ioei_interrupt() description

Message-id: <1410545655-205645-7-git-send-email-dzickus@redhat.com>
Patchwork-id: 94064
O-Subject: [RHEL7 PATCH 006/626] trivial: powerpc: Fix typo in ioei_interrupt() description
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit e1b85c17bf3e4f2ecbf9ec824c4048a06078100b
Author: Sebastien Bessiere <sebastien.bessiere@gmail.com>
Date:   Fri Jun 14 17:57:03 2013 +0200

    trivial: powerpc: Fix typo in ioei_interrupt() description

    Signed-off-by: Sebastien Bessiere <sebastien.bessiere@gmail.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/pseries/io_event_irq.c b/arch/powerpc/platforms/pseries/io_event_irq.c
index ef9d9d8..5ea88d1 100644
--- a/arch/powerpc/platforms/pseries/io_event_irq.c
+++ b/arch/powerpc/platforms/pseries/io_event_irq.c
@@ -115,7 +115,7 @@ static struct pseries_io_event * ioei_find_event(struct rtas_error_log *elog)
  *   by scope or event type alone. For example, Torrent ISR route change
  *   event is reported with scope 0x00 (Not Applicatable) rather than
  *   0x3B (Torrent-hub). It is better to let the clients to identify
- *   who owns the the event.
+ *   who owns the event.
  */
 
 static irqreturn_t ioei_interrupt(int irq, void *dev_id)
-- 
1.7.1